recipes/metall/all/conanfile.py

```diff
@@ -66,9 +26,15 @@ def _compilers_minimum_version(self):
             "clang": "9",
         }
 
+    def requirements(self):
+        self.requires("boost/1.79.0")
+
+    def package_id(self):
+        self.info.header_only()
```

Comment:
```suggestion
        self.info.clear()
```
Available since 1.50.0: https://docs.conan.io/en/latest/changelog.html#jun-2022

https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#the-package-id-method